src/main/kotlin/uk/gov/justice/digital/hmpps/externalusersapi/resource/UserGroupController.kt

```diff
@@ -0,0 +1,94 @@
+package uk.gov.justice.digital.hmpps.externalusersapi.resource
+
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.Parameter
+import io.swagger.v3.oas.annotations.media.Content
+import io.swagger.v3.oas.annotations.media.Schema
+import io.swagger.v3.oas.annotations.responses.ApiResponse
+import io.swagger.v3.oas.annotations.responses.ApiResponses
+import io.swagger.v3.oas.annotations.tags.Tag
+import org.slf4j.LoggerFactory
+import org.springframework.http.HttpStatus
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.web.bind.annotation.DeleteMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.ResponseStatus
+import org.springframework.web.bind.annotation.RestController
+import uk.gov.justice.digital.hmpps.externalusersapi.config.ErrorResponse
+import uk.gov.justice.digital.hmpps.externalusersapi.service.UserGroupService
+
+@RestController
+@Tag(name = "/user/id/{userId}/groups", description = "User Groups Controller")
+class UserGroupController(
+  private val userGroupService: UserGroupService
+) {
+
+  companion object {
+    private val log = LoggerFactory.getLogger(this::class.java)
+  }
+
+  @DeleteMapping("/user/id/{userId}/groups/{group}")
+  @ResponseStatus(HttpStatus.NO_CONTENT)
```

Comment:
I think this should really be /users (rather than /user) as that seems to be a standard

Reply:
Done